### PR TITLE
Created search_in_sorted_rotated_array.py

### LINKED ISSUE #51: Search in Sorted Rotated Array
Given an integer array, and an element x, find the index of the element in O(log n) time.
Integer array is sorted in ascending order, but it is rotated at some pivot.
Example: 
Input: arr[] = {5, 6, 7, 8, 9, 10, 1, 2, 3}, x = 3
Output: 8

### PR DESCRIPTION
**Pull request by sg-iitg**
Fixes #51

### CODE REVIEW

**Comment by sg-iitg** [comment]
@plazzy99 Please review my Pull request linked to issue#361

**Comment by Rohan27s** [comment]
@sg-iitg Resolve conflicts

**Comment by sg-iitg** [comment]
Done! @Rohan27s

**Review by Rohan27s — APPROVED** [review]

File: Code/Python/readme.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
- # Codes in the Python language
+# Codes in the Python language
 
- Tower of Hanoi ---> [Python] (Code/Python/Tower_of_Hanoi.py)
\ No newline at end of file
+ Tower of Hanoi ---> [Python] (Code/Python/Tower_of_Hanoi.py)
```
Comment (Rohan27s) [inline Code/Python/readme.md:3]: Update readmme

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/Python/readme.md (hunk 1 above)
Comment (sg-iitg) [reply Code/Python/readme.md]: Is it fine now ?

File: Code/Python/readme.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
- # Codes in the Python language
+# Codes in the Python language
 
- Tower of Hanoi ---> [Python] (Code/Python/Tower_of_Hanoi.py)
```
Comment (Rohan27s) [inline Code/Python/readme.md:3]: Remove the space between [Python] add the parenthesis to make it work(same goes for the other line)

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/Python/readme.md (hunk 2 above)
Comment (sg-iitg) [reply Code/Python/readme.md:3]: Done

Comment (Rohan27s) [reply Code/Python/readme.md:3]: Wait..why have you make changes in this readme file of python/readme....you have make this changes in Algo-Tree/Array/readme.md

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/Python/readme.md (hunk 2 above)
Comment (sg-iitg) [reply Code/Python/readme.md:3]: Oh, sorry I was biased by one of the code linking present in this readme related to Tower of Hanoi code.

Comment (sg-iitg) [reply Code/Python/readme.md:3]: @Rohan27s The tower of hanoi linking is also incorrect, so I have removed that from the readme, I hope thats fine

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**